tests/group_opacity_reuse_picosvg.ttx

```diff
@@ -61,8 +61,8 @@
     <svgDoc endGlyphID="2" startGlyphID="2">
       <![CDATA[<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1">
   <g id="glyph2" transform="matrix(0.781 0 0 0.781 0 -100)">
-    <path d="M60,10 L80,10 L80,60 L60,60 L60,10 Z" id="e000.0"/>
-    <g opacity="0.6">
+    <path d="M60,10 L80,10 L80,60 L60,60 L60,10 Z" fill="black" id="e000.0"/>
+    <g fill="black" opacity="0.6">
```

Comment:
is "black" the shortest way to spell black? Is 3-digit hex legal, e.g. #000? ... maybe that's not enough difference to bother lol